.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please drop this file from the PR. We don't need `.vscode` stuff in the codebase.

Reply:
Thanks for noticing.  I'm usually super clean about that... but this once slipped by.